src/main/java/org/examplesolid/domain/model/api/CharacterAPI.java

```diff
@@ -10,6 +10,10 @@ public record CharacterAPI(int id,
                            Location origin,
                            Location location,
                            String image,
-                           List<String> episodes,
+                           List<String> episode,
                            String url, String created) {
+
+    public CharacterAPI(String name) {
+        this(0, name, null, null, null, null, null, null, null, null, null);
```

Comment:
Cual es el proposito the esta linea? 
por que se debe colocar null? 
Quizas hay una forma de hacer nullable las propiedades. 🤔